com.unity.render-pipelines.universal/Runtime/2D/Renderer2D.cs

```diff
@@ -49,104 +65,144 @@ public Renderer2DData GetRenderer2DData()
             return m_Renderer2DData;
         }
 
-        public override void Setup(ScriptableRenderContext context, ref RenderingData renderingData)
+        void CreateRenderTextures(
+            ref CameraData cameraData,
+            bool forceCreateColorTexture,
+            FilterMode colorTextureFilterMode,
+            CommandBuffer cmd,
+            out RenderTargetHandle colorTargetHandle,
+            out RenderTargetHandle depthTargetHandle)
         {
-            ref CameraData cameraData = ref renderingData.cameraData;
             ref var cameraTargetDescriptor = ref cameraData.cameraTargetDescriptor;
-            PixelPerfectCamera ppc;
-            cameraData.camera.TryGetComponent<PixelPerfectCamera>(out ppc);
-
-            Vector2Int ppcOffscreenRTSize = ppc != null ? ppc.offscreenRTSize : Vector2Int.zero;
-            bool ppcUsesOffscreenRT = ppcOffscreenRTSize != Vector2Int.zero;
-            bool postProcessEnabled = renderingData.cameraData.postProcessEnabled;
-            bool useOffscreenColorTexture =
-                ppcUsesOffscreenRT || postProcessEnabled || cameraData.isHdrEnabled || cameraData.isSceneViewCamera || !cameraData.isDefaultViewport || !m_UseDepthStencilBuffer;
-
-            // Pixel Perfect Camera may request a different RT size than camera VP size.
-            // In that case we need to modify cameraTargetDescriptor here so that all the passes would use the same size.
-            if (ppcUsesOffscreenRT)
+
+            if (cameraData.renderType == CameraRenderType.Base)
+            {
+                m_CreateColorTexture = forceCreateColorTexture
```

Comment:
We should probably add some helper function to check these flags like to create an intermediate render texture in future as well